docs/sources/mobile-app/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Grafana OnCall mobile app
+aliases:
+  - /docs/oncall/latest/mobile-app/
+keywords:
+  - Mobile App
+  - oncall
+  - notification
+  - push notification
+weight: 1200
+---
+
+# Grafana OnCall Mobile App
+
+>**Note**: This application is currently in beta
+
+The Grafana OnCall mobile app allows teams to monitor and respond to critical system events from anywhere.
+On-call engineers can start exploring the benefits of the Grafana OnCall mobile app, with real-time alerts, on-call notifications, and a growing feature set.
+
+
+## About Grafana OnCall mobile app
+Grafana OnCall mobile app key features:
+
+- Override Do Not Disturb on your device to ensure delivery of critical alerts
+- Receive push notifications according to your personal notification settings
+- View alert details from your mobile device
+- Login with a quick and secure QR code authorization
+
+The OnCall mobile app allows users to receive push notifications as part of their notification policy.
+Push notifications are one of the many notification options in Grafana OnCall.
+Consider using multiple notification methods, such as mobile app and phone/SMS, to remain available in case of degraded internet or phone network connectivity.
+
+The current version of the mobile app includes limited functionality and serves primarily as a notification method. The mobile app is in development, and some features are not yet available.
+Stay tuned, we're working on it!
+
+## Before you begin
+
+The Grafana OnCall mobile app is intended to be used as an extension of your Grafana OnCall instance.
+Grafana OnCall is available for Grafana Cloud and Grafana open source users.
+
+- You must have a Grafana OnCall account to use this application
+- Download Grafana OnCall mobile app
+
+Mobile app download - [Google Play Store](https://play.google.com/apps/testing/com.grafana.oncall.prod)
+Grafana OnCall for iOS coming soon.
+
+## Connect your Grafana OnCall account 
+
+The OnCall mobile app uses a QR code authentication to connect to your Grafana OnCall instance.
+You can associate one Grafana OnCall user with your OnCall mobile app.
+
+To connect your account in the mobile app:
+1. Open the Grafana OnCall mobile app
+2. Click **Sign in** and follow the instructions in the app to locate your QR code
+4. Use the mobile app to scan the QR code in your User profile
```

Comment:
"in your User profile" is a bit confusing here as it's the first time we mention it, people may think it refers to their profile within the app, which doesn't exist yet. I would just remove these words as we already said to follow instructions in the app to find where the QR code is.

anyway, this is maybe nitpicking. so do as you like with this feedback, let's get the docs out :)

Reply:
All feedback, including nits, is welcome and helpful! I appreciate you taking the time to consider the more subtle details and share your perspective. 

I went ahead and removed that step to simplify the guidance. As you said, the app will get users through these steps anyways.